### PR TITLE
Minor UI improvements to WiX BA.

### PR DESCRIPTION
**Pull request by robmen**
Show the package currently being installed and trim off irrelevant noise
from the beginning of the progress messages.

### CODE REVIEW

File: src/Setup/WixBA/Model.cs [hunk 1]
```diff
@@ -67,21 +92,7 @@ public Model(BootstrapperApplication bootstrapper)
...
-                    Assembly assembly = Assembly.GetExecutingAssembly();
-                    FileVersionInfo fileVersion = FileVersionInfo.GetVersionInfo(assembly.Location);
-
-                    this.version = new Version(fileVersion.FileVersion);
-                }
-
-                return this.version;
-            }
-        }
+        public Version Version { get; set; }
```
Comment (barnson) [inline src/Setup/WixBA/Model.cs:95]: private set?

File: src/Setup/WixBA/RootView.xaml [hunk 2]
```diff
@@ -68,7 +68,10 @@
 
             <!-- Progress -->
             <ProgressBar Grid.Row="1" Grid.ColumnSpan="3" Margin="3" DataContext="{Binding ProgressViewModel}" Value="{Binding Path=Progress}" Visibility="{Binding ProgressEnabled, Converter={StaticResource BooleanToVisibilityConverter}}"/>
-            <TextBlock Grid.Row="1" Grid.ColumnSpan="3" Style="{StaticResource StatusTextStyle}" DataContext="{Binding ProgressViewModel}" Visibility="{Binding ProgressEnabled, Converter={StaticResource BooleanToVisibilityConverter}}" Text="{Binding Message}"/>
+            <StackPanel Grid.Row="1" Grid.ColumnSpan="3">
+                <TextBlock Style="{StaticResource StatusTextStyle}" DataContext="{Binding ProgressViewModel}" Visibility="{Binding ProgressEnabled, Converter={StaticResource BooleanToVisibilityConverter}}" Text="{Binding Package}"/>
+                <TextBlock Style="{StaticResource StatusTextStyle}" DataContext="{Binding ProgressViewModel}" Visibility="{Binding ProgressEnabled, Converter={StaticResource BooleanToVisibilityConverter}}" Text="{Binding Message}"/>
```
Comment (barnson) [inline src/Setup/WixBA/RootView.xaml:73]: Pretty.

**Comment by barnson** [comment]
History?

File: src/Setup/WixBA/Model.cs (hunk 1 above)
Comment (robmen) [inline src/Setup/WixBA/Model.cs:95]: Yeah, could definitely do that.

**Comment by robmen** [comment]
Small cosmetic change, didn't seem worth noting in history.

**Pull request merged**